### PR TITLE
bugfix/Audio Manager Null

### PR DESCRIPTION
**Pull request by DJSteins**
Basically refactors a couple of audio manager methods so they always require a null check first.

### CODE REVIEW

File: Assets/_PartyParkade/Sprites/Obstacles/Materials/Parking.mat [hunk 1]
```diff
@@ -0,0 +1,76 @@
+%YAML 1.1
```
Comment (m4rtint) [inline Assets/_PartyParkade/Sprites/Obstacles/Materials/Parking.mat:1]: you changedd the mat?

Comment (DJSteins) [reply Assets/_PartyParkade/Sprites/Obstacles/Materials/Parking.mat:1]: oops

Comment (DJSteins) [reply Assets/_PartyParkade/Sprites/Obstacles/Materials/Parking.mat:1]: It looks like it created a new file when I added the parking spot to the flat map?

Comment (DJSteins) [reply Assets/_PartyParkade/Sprites/Obstacles/Materials/Parking.mat:1]: Should I just delete them?

Comment (m4rtint) [reply Assets/_PartyParkade/Sprites/Obstacles/Materials/Parking.mat:1]: yes please. very styrange

File: Assets/_PartyParkade/Script/PowerUp/OilSpill.cs [hunk 2]
```diff
@@ -90,6 +93,9 @@ override protected void HandleTrigger(Collider other)
         m_carsToRotate.Add(car);
         m_carVelocities.Add(car.CarRigidbody.velocity);
 
-        PlaySFX(AudioManager.PowerUpsSFX.OilSpillSlip);
+        if (AudioManager.Instance)
+        {
+            PlaySFX(AudioManager.Instance.PowerUpsSFX.OilSpillSlip);
```
Comment (m4rtint) [inline Assets/_PartyParkade/Script/PowerUp/OilSpill.cs:98]: How come you don't set the audiogroup for this sfx?

Comment (m4rtint) [reply Assets/_PartyParkade/Script/PowerUp/OilSpill.cs:98]: and you can probably abstract this into its own method

**Comment by mikeslgoh** [comment]
[AudioManager](https://trello.com/c/4plduwKb/541-audiomanager)

**Pull request merged**